pkgs/tools/misc/diffoscope/default.nix

```
@@ -35,6 +35,8 @@ python3Packages.buildPythonApplication rec {
 
     # When generating manpage, use the installed version
     substituteInPlace doc/Makefile --replace "../bin" "$out/bin"
+
+    sed -e 's/  Stream/Stream/g' -i tests/data/mp3_expected_diff
```

Please use substituteInPlace because sed will silently fail.

---

Fixed!